src/util/address.ts

```diff
@@ -1,8 +1,8 @@
 /**
- * @module util/address
+ * @module util/addressbool
```

Comment:
this should probably be just address
```suggestion
 * @module util/address
```